### PR TITLE
Provide meaningful string names to Cylinder inputs

### PR DESCRIPTION
**Pull request by sytuannguyen**
Cylinder geometry will be extensively considered in the near future for wellbore problems. This tiny PR provides meaningfull strings to Cylinder inputs as follow:

`point1` is changed to `lowerFaceCenter`
`point2` is changed to `upperFaceCenter`

This is only changed on the user side for instant. This will help the users to better understand the input data in xml files.
Rebaseline is not needed for this PR.

Rebaseline done in https://github.com/GEOSX/integratedTests/pull/312

### CODE REVIEW

**Review by jhuang2601 — APPROVED** [review]

File: src/coreComponents/mesh/simpleGeometricObjects/Cylinder.hpp [hunk 1]
```diff
@@ -72,8 +72,8 @@ class Cylinder : public SimpleGeometricObjectBase
 
   struct viewKeyStruct
   {
-    static constexpr char const * point1String() { return "point1"; }
-    static constexpr char const * point2String() { return "point2"; }
+    static constexpr char const * point1String() { return "lowerFaceCenter"; }
+    static constexpr char const * point2String() { return "upperFaceCenter"; }
```
Comment (francoishamon) [inline src/coreComponents/mesh/simpleGeometricObjects/Cylinder.hpp:76]: @sytuannguyen Are you sure that there is the concept of "lower" and "upper" face in the `Cylinder` class? Upper and lower with respect to what?

Comment (sytuannguyen) [reply src/coreComponents/mesh/simpleGeometricObjects/Cylinder.hpp:76]: @francoishamon You are right. How about firstFaceCenter and secondFaceCenter?

Comment (francoishamon) [reply src/coreComponents/mesh/simpleGeometricObjects/Cylinder.hpp:76]: Yes, I think this is better

Comment (sytuannguyen) [reply src/coreComponents/mesh/simpleGeometricObjects/Cylinder.hpp]: done

File: src/coreComponents/mesh/simpleGeometricObjects/Cylinder.cpp [hunk 2]
```diff
@@ -35,15 +35,15 @@ Cylinder::Cylinder( const string & name, Group * const parent ):
...
 
   registerWrapper( viewKeyStruct::point2String(), &m_point2 ).
     setInputFlag( InputFlags::REQUIRED ).
-    setDescription( "Center point of the other face of the cylinder" );
+    setDescription( "Center point of the second face of the cylinder" );
 
   registerWrapper( viewKeyStruct::radiusString(), &m_radius ).
     setInputFlag( InputFlags::REQUIRED ).
-    setDescription( "Radius of the cylinder" );
+    setDescription( "Radius of the cylinder, outer radius of the annulus" );
```
Comment (francoishamon) [inline src/coreComponents/mesh/simpleGeometricObjects/Cylinder.cpp:46]:
```suggestion
    setDescription( "Outer radius of the cylinder" );
```

**Review by francoishamon — APPROVED** [review]

File: src/coreComponents/mesh/simpleGeometricObjects/Cylinder.cpp (hunk 2 above)
Comment (sytuannguyen) [reply src/coreComponents/mesh/simpleGeometricObjects/Cylinder.cpp]: done

**Pull request merged**